docs/docs/en/guides/develop/registry/continuous-integration.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Continuous integration
+titleTemplate: :title · Build · Develop · Registry · Continuous integration
+description: Learn how to use the Tuist Registry in continuous integration.
+---
+
+# Continuous Integration (CI) {#continuous-integration-ci}
+
+To use the registry on your CI, you need to ensure that you have logged in to the registry by running `tuist registry login` as part of your workflow.
```

Comment:
Something that comes to my mind every time I see `tuist registry login` necessary on CI is that I'd expect exposing a token in CI to be sufficient to be authenticated. The steps contained in `tuist registry login` as needed, so I wonder if we can have a flag in other commands to do it as part of their business logic:

```bash
tuist generate --no-registry-authentication
tuist generate --registry-authentication
```

No need to action on this, but since I saw the documentation about it, I thought this warrants some thinking to see if we can improve the DX.

Reply:
We _could_ do that, but I'd love to keep the flows for setups with generated projects and without them as similar as possible unless there would be a strong reason/DX win.